violet-framework/src/main/java/com/horstmann/violet/workspace/editorpart/behavior/ShowMenuOnRightClickBehavior.java

```diff
@@ -72,6 +72,20 @@ private JPopupMenu getPopupMenu() {
      */
     private JPopupMenu fillMenu(JPopupMenu aPopupMenu)
     {
+    	select.addActionListener(new ActionListener(){
+    		public void actionPerformed(ActionEvent event)
+    		{
+    			IEditorPartBehaviorManager behaviorManager = ShowMenuOnRightClickBehavior.this.editorPart.getBehaviorManager();
+    			List<SelectSelectToolBehavior> found = behaviorManager.getBehaviors(SelectSelectToolBehavior.class);
+    			
+                if (found.size() != 1) {
+                    return;
```

Comment:
I think, it isn't a good practise, but it's only my opinion. 
https://stackoverflow.com/questions/9003283/what-is-the-correct-way-to-end-a-void-function
https://stackoverflow.com/questions/1283325/is-it-bad-practice-to-use-return-inside-a-void-method